client/src/components/ratings-reviews/ReviewItem.jsx

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable react/jsx-one-expression-per-line */
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+// import ReviewPics.jsx from './ReviewPics.jsx';
 
 class ReviewItem extends React.Component {
   constructor(props) {
     super(props);
-    console.log('props', props);
   }
 
   render(props) {
```

Comment:
This doesn't need to be done now, but in the future according to Airbnb style guides, it will ask you destructure your props. So in the future, you could destructure props by doing: 
`const { review } = this. props`
`const { reviewer_name, date, review_id } = review
Then for each p div you could write it as:
`Name: {reviewer_name}`
etc